quicklogic/qlf_k4n8/tests/design_flow/CMakeLists.txt

```diff
@@ -9,7 +9,6 @@ add_subdirectory(and2_latch)
 add_subdirectory(bin2bcd)
 add_subdirectory(clock_tree_design)
 add_subdirectory(full_adder)
-add_subdirectory(io_reg)
```

Comment:
Any reason why this one is removed?

Reply:
Possibly, this is a rebase error, I'll add this one back

Reply:
Actually, the `io_reg` test is tested in the `features` test, and this was actually removed downstream as well.